procrustes/test/test_psdp.py

```diff
@@ -24,16 +24,19 @@
 
 import numpy as np
 from numpy.testing import assert_almost_equal
-from procrustes.psdp import psdp_woodgate
+from procrustes.psdp import psdp_peng, psdp_woodgate
 
 
 def test_psdp_identity(n=np.random.randint(50, 100)):
     r"""Test PSDP with identity matrix."""
     a = np.eye(n)
     b = np.eye(n)
     res = psdp_woodgate(a=a, b=b)
-    s = res["s"]
-    error = res["error"]
+    s, error = res["s"], res["error"]
+    assert_almost_equal(s, np.eye(n))
+    assert_almost_equal(error, 0.0)
+    res = psdp_peng(a=a, b=b)
+    s, error = res["s"], res["error"]
     assert_almost_equal(s, np.eye(n))
     assert_almost_equal(error, 0.0)
```

Comment:
Splitting this testing function into two is a better choice as it would be easier for the other developers to know which part is breaking. This makes debugging easier. So I would split `test_psdp_identity` into `test_psdp_woodgate_identity` and `test_psdp_peng_identity`.

Reply:
Yeah, I have fixed all of that in [8c7015b](https://github.com/theochem/procrustes/pull/187/commits/8c7015b5413c07718ecf7cd817f58568144fa3ae)